config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-periodics.yaml

```diff
@@ -45,6 +45,14 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-openstack-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "2"
     testgrid-alert-stale-results-hours: "48"
+  reporter_config:
```

Comment:
What's the benefit of running this as a periodic when we also run it as a presubmit? Could we just remove this job?

Reply:
My understanding of postsubmit is that unless the merge is a fast forward we run the tests again against the merged commit. If that's correct, I think we have to keep the postsubmit test as merges are a genuine source of errors.

If we wanted to cut one of these I'd make it optional as a presubmit but required as a postsubmit. We could also look into limiting the scope of changes that it runs against, although I don't think this will save us much in practise.

Reply:
I don't think our dependencies should change, should they? Is there any dependency we don't pin?

Reply:
You are probably right. There could be some things in the devstack scripts that are not pinned but not really dependencies.

Good point about the postsubmit also. It is too bad that it is so "hidden" though. I'm not sure how we would notice if the postsubmit fails. There are no alerts for them so most likely I would see it only a couple of days later when the periodic test sends out the email about 2 failures in a row.
The presubmit feels like a nice safety check though. It should catch most issues but as you say, there could still be issues with the merge even when the PR commit itself was passing.

Reply:
I suspect I'm confused about post-submit jobs, btw. I suspect the second job run is more likely a feature of pre-submits. I've asked for clarification in #sig-testing because I can't find it in the docs.

Reply:
Based on the [slack discussion](https://kubernetes.slack.com/archives/C09QZ4DQB/p1694423657238009), I think we're saying we should run the regular e2e tests only as a presubmit? Specifically we should remove the post-submit and periodic runs?

We should run e2e-full as a periodic because we don't run it as a presubmit.

Reply:
I mostly agree, though I can see some value in running the normal e2e as periodic, can we keep that? So remove e2e post-submit, add e2e-full periodic

Reply:
Done!